templates/download/intel-nuc.html

```diff
@@ -1,135 +1,196 @@
+            <li class="p-list__item">Select the USB flash drive as a boot option.</li>
+            <li class="p-list__item">Select "Try Ubuntu without installing".</li>
+          </ol>
+        </div>
+      </li>
+      <li class="p-stepped-list__item">
+        <hr class="p-rule u-hide--large" />
+        <h2 class="p-stepped-list__title">
+          Flash Ubuntu Core
+        </h2>
+        <div class="p-stepped-list__content">
+          <ol class="p-list--divided u-no-margin--left">
+            <li class="p-list__item">Once the Ubuntu session has started, insert the second USB flash drive containing the Ubuntu Core image file.</li>
+            <li class="p-list__item">Open a terminal and use the following command to find out the target disk device to install the Ubuntu Core image to:<br />
+            <pre><code>sudo fdisk -l</code></pre></li>
+            <li class="p-list__item">Run the following command, where <code>&lt;disk label&gt;</code> is the label of the second USB flash drive:<br />
+            <pre><code>xzcat /media/ubuntu/&lt;disk label&gt;/dawson-uc18-m7-20190122-10.img.xz | sudo dd of=/dev/&lt;target disk device&gt; bs=32M status=progress; sync</code></pre></li>
+            <li class="p-list__item">Reboot the system and remove the flash drives when prompted. It will then boot from the internal memory where Ubuntu Core has been flashed.</li>
+          </ol>
+        </div>
+      </li>
+      <li class="p-stepped-list__item">
+        <hr class="p-rule u-hide--large" />
+        <h2 class="p-stepped-list__title">
+          First boot setup
+        </h2>
         <div class="p-stepped-list__content">
-          <ol class="u-no-margin--left">
-            <li>Start the NUC and push F10 to enter the boot menu.</li>
-            <li>Select the USB flash drive as a boot option.</li>
-            <li>Select "Try Ubuntu without installing".</li>
+          <ol class="p-list--divided u-no-margin--left">
+            <li class="p-list__item">The system will boot then become ready to configure.</li>
+            <li class="p-list__item">The device will display the prompt "Press enter to configure".</li>
+            <li class="p-list__item">Press enter then select "Start" to begin configuring your network and an administrator account. Follow the instructions on the screen, you will be asked to configure your network and enter your Ubuntu SSO credentials.</li>
+            <li class="p-list__item">
+              <p>At the end of the process, you will see your credentials to access your Ubuntu Core machine:</p>
+              <pre><code>This device is registered to &lt;Ubuntu SSO email address&gt;.
+      Remote access was enabled via authentication with the SSO user &lt;Ubuntu SSO user name&gt;
+      Public SSH keys were added to the device for remote access.</code></pre>
+            </li>
           </ol>
         </div>
       </li>
```

Comment:
Same thing here, there's a partial for this already with exactly the same code `/download/shared/_first-boot-setup.html`